fake_ubersmith/api/methods/client.py

```diff
@@ -50,6 +50,18 @@ def hook_to(self, entity):
             ubersmith_method='client.get',
             function=self.client_get
         )
+        entity.register_endpoints(
+            ubersmith_method='client.add',
+            function=self.client_add
+        )
+
+    def client_add(self, form_data):
+        client_id = len(self.clients)
+
+        form_data["clientid"] = client_id
```

Comment:
If we want to change client_id to clientid I think we should do it another PR since it would be a different issue category.